_posts/2020-12-16-how-to-update-to-terraform-13.adoc

```diff
@@ -225,14 +225,14 @@ Terraform 0.13. You can refer to the pull requests to see an example of the upda
 Reference Architecture work with Terraform 0.13:
 
 Standard Reference Architecture::
-  See the https://github.com/gruntwork-io/infrastructure-modules-multi-account-acme/pull/46[infrastructure-modules PR]
```

Comment:
NIT: perhaps include migration instructions and both PRs? i.e., Guide + 2 examples?